flight/targets/sprf3e/fw/pios_config.h

```diff
@@ -0,0 +1,96 @@
+/**
+ ******************************************************************************
+ * @addtogroup Targets Target Boards
+ * @{
+ * @addtogroup spf3e SP Racing F3E support files
+ * @{
+ *
+ * @file       sprf3e/board-info/pios_config.h
+ * @author     Tau Labs, http://taulabs.org, Copyright (C) 2012-2015
+ * @author     dRonin, http://dRonin.org/, Copyright (C) 2016
+ * @brief      Board specific options that modify PiOS capabilities
+ * @see        The GNU Public License (GPL) Version 3
+ *
+ *****************************************************************************/
+/*
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, see <http://www.gnu.org/licenses/>
+ */
+
+#ifndef PIOS_CONFIG_H
+#define PIOS_CONFIG_H
+
+#include <pios_flight_config.h>
+
+/* Major features */
+
+/* Enable/Disable PiOS Modules */
+#define PIOS_INCLUDE_DMA_CB_SUBSCRIBING_FUNCTION
+#define PIOS_INCLUDE_I2C
+#define PIOS_INCLUDE_SPI
+#define WDG_STATS_DIAGNOSTICS
+
+/* Select the sensors to include */
+#define PIOS_INCLUDE_MPU9250_SPI
+//#define PIOS_MPU6000_ACCEL
+#define PIOS_INCLUDE_BMP280
+#define PIOS_INCLUDE_HMC5883
+#define PIOS_INCLUDE_HMC5983_I2C
+#define PIOS_TOLERATE_MISSING_SENSORS
```

Comment:
BMP280 is placed on original SP Racing F3 Evo and XSRF3E. Where are clone boards without baro, and most of the brushed version clones do not have it.
HMC is as option for external mag. Clone boards use sometimes a MPU without internal mag. What's why I implemented the option for external mag. Also sometimes it is better, to place a GPS/Mag combo away from the other electronics. Original SP  Racing F3 Evo (and 1:1 clones) has I2C on separate connector, so it is easy to connect external mag. I tested external mag with HMC5883 breakout board.